manifests/p/pnpm/pnpm/8.14.0/pnpm.pnpm.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with komac v2.0.4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: pnpm.pnpm
+PackageVersion: 8.14.0
+InstallerType: exe

```

```suggestion
InstallerType: portable
```

I am fairly certain that pnpm executables are in fact not actual installers but simple, self-contained applications that should have their `InstallerType` set to `portable`.